GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -487,7 +487,7 @@ extension CameraViewController {
      - parameter qrDocument: A validated `GiniQRCodeDocument`.
      - parameter didTapDone: Block executed when the user taps the _Done_ button to proceed with the `GiniQRCodeDocument`.
      */
-    func showPopup(forQRDetected qrDocument: GiniQRCodeDocument, didTapDone: @escaping () -> Void) {
+    public func showPopup(forQRDetected qrDocument: GiniQRCodeDocument, didTapDone: @escaping () -> Void) {
```

Comment:
If you make the popup part of CameraViewController's internal responsibilities, you'd be able to remove this `public`. Which would be awesome since it's a confusing part of the public interface. What popup? Why?

Reply:
Answered above.